jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -421,8 +440,50 @@ public void onClick( ClickEvent event ) {
 
         controlBulkOperations();
     }
+    
+    private Column<ProcessInstanceSummary, ProcessInstanceSummary> initErrorCountColumn() {
+        
+        Column<ProcessInstanceSummary, ProcessInstanceSummary> column = new Column<ProcessInstanceSummary, ProcessInstanceSummary>(
+                new AbstractCell<ProcessInstanceSummary>(){
+
+            @Override
+            public void render(Context context, ProcessInstanceSummary value, SafeHtmlBuilder sb) {
+                int errCount =  value != null ? value.getErrorCount() : 0;
+                ProcessRuntimeCSS css = ProcessRuntimeResources.INSTANCE.css();
+                //TODO: set link URL to correct error view perspective path
+                String linkToErrorView = "#Error View?processInstanceId=" + value.getProcessInstanceId();
+                String cssClass = "label-" + (errCount > 0 ? "danger" : "default");
+                String elementId = DOM.createUniqueId();
+                sb.appendHtmlConstant("<a id=\"" + elementId + "\" href=\"#\" class=\"label " + css.processInstanceErrorCount() +
```

Comment:
@rbarriuso if you use `badge` class all you need is to set a different background color when you have errors: `background-color: #c00;`
Apart from that, `badge` introduces an unnecessary margin that you can remove using: `margin-left: 0px;`